woocommerce/woocommerce-ios
# Add view events to delete tracking button in Order Details and Fulfill Order

Fixes #994 

This PR is against the `release/1.9` branch

Add the following view events:
```
*_order_fulfillment_tracking_delete_button_tapped
*_order_detail_tracking_delete_button_tapped
```

according to the discussion in [this thread](https://wp.me/p99K0U-1eo-p2)

## Testing
- Checkout the branch. 
- Delete a shipment tracking from the Order Details screen. Check that the Tracks event `order_detail_tracking_delete_button_tapped` is submitted
- Delete a shipment tracking from the Fulfill Order screen. Check that the Tracks event `order_fulfillment_tracking_delete_button_tapped`  is submitted
